core/src/storage2/collections/binary_heap/elements.rs

```diff
@@ -43,7 +40,7 @@ use crate::storage2::{
 /// the `BinaryHeap` this interface transposes heap indices to the child inside
 /// the `Children` object, in which the element is stored.
 #[derive(Default, PartialEq, Eq, Debug)]
-pub struct Elements<T>
+pub(crate) struct Elements<T>
```

Comment:
I bet you could even make this public only to the `binary_heap` module instead of the crate.

Reply:
Same for all the other `pub(crate)` entities.

Reply:
Also still need to find a better name for this: `PairedChildren` maybe?